source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -244,6 +246,25 @@ To move data between two PaaS-hosted PostgreSQL databases:
 
     where `DESTINATION_SERVICE_NAME` is the name of the target database.
 
+    You will see some error messages when importing the data file into the target database. [These errors are expected](/deploying_services/postgresql/#import-data-expected-errors) and the import process will succeed.
+
+<h4 id="import-data-expected-errors">Expected Errors</h4>
+
+When importing data into a target database you may encounter errors which do not stop the import process from finishing:
```

Comment:
```suggestion
When importing data into a target database, you may encounter errors that do not stop the import process from finishing:
```

Reply:
I have made the suggested change to the document.